### ISSUE TITLE
[Bug]: Swarmer Deconstruction does not create plating under walls or deconstructs plating under walls

### ISSUE THREAD

**Issue by Programs-The-Station** [BeeStation/BeeStation-Hornet#10223]
### Round Date

2023/11/24

### Round ID

46726

### Describe the bug

We had swarmers on station.

I was in atmospherics, when one punched through the reinforced wall in the distro room from the AI tube room. When the R Wall went away, there was no plating underneath the wall. This spaced the two areas. There were multiple callouts of this happening across the station.

I'm guessing this has to do with #9503. Might be fixed with #10219, might not be.

Could this be caused by the plating underneath the wall getting one of the "weakened" plating types (rusted, etc), so the final deconstruct pushes it over the limit?

### What did you expect?

Swarmers should not space areas by deconstructing walls.

### Reproducible steps

1. Become swarmer.
2. Eat valid R Wall (may happen with normal walls).
3. Plating may or may not be generated. I don't think it happened every time.

**Comment by HowToLoLu** [comment]
Got some testing done and confirmed it.

https://github.com/BeeStation/BeeStation-Hornet/assets/22382345/25f591dd-d3b7-4e60-9f98-bbdd25d50e57

It apparently only is an issue with R-walls. Also found that occasionally there will be no girder left behind, and the plating will be damaged

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/22382345/972d44a9-7d0d-434b-a2ff-80fe61c56ac5)

**Comment by PowerfulBacon** [comment]
Okay for some reason swarmer attacks put things into the explosions subsystem

**Comment by Miliviu** [comment]
can't really replicate it...


https://github.com/BeeStation/BeeStation-Hornet/assets/45698448/ff7e6bee-cfef-4553-8400-9187d3aed49c

**Comment by HowToLoLu** [comment]
Yeah it was likely fixed and whoever fixed it never linked the issue
